src/code42cli/cmds/legal_hold.py

```diff
@@ -124,6 +138,37 @@ def show(state, matter_id, include_inactive=False, include_policy=False):
         echo("")
 
 
+@legal_hold.command()
+@click.option(
+    "--matter-id", type=str, help="Filter results by legal hold UID",
```

Comment:
Small refactor opportunity: we have matter_id_option already defined in this module.  We could create a shared method

```
def matter_id_option(required, help):
  return click.option(
    "-m",
    "--matter-id",
    required=required,
    type=str,
    help=help,
)
```

And use that for all matter_id options and then only have set two fields